src/main/kotlin/com/terraformation/backend/documentproducer/db/DocumentStore.kt

```diff
@@ -101,7 +101,7 @@ class DocumentStore(
     return ExistingSavedVersionModel(versionsRow)
   }
 
-  fun findAll(): List<ExistingDocumentModel> = fetchByCondition()
+  fun fetchAll(): List<ExistingDocumentModel> = fetchByCondition()
```

Comment:
This was renamed to be consistent with other functions that are fetching things from the database using the DSL context as opposed to the DAO classes.